### PR TITLE
Awesome books es6 branch

### PR DESCRIPTION
**Pull request by sulaimonaa**
#Awesome book with es6
##Uses javascript classes with module
##All other script files are imported to the index.js file
### file workflow structures include index.html, index.css, and index.js

### CODE REVIEW

File: modules/changeState.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  bookDisplay.style.display = 'none';
+  formDisplay.style.display = 'block';
+  contactDisplay.style.display = 'none';
+}
+
+export function displayContact() {
+  bookDisplay.style.display = 'none';
+  formDisplay.style.display = 'none';
+  contactDisplay.style.display = 'block';
+}
```
Comment (codepantha) [inline modules/changeState.js]: - [ ] Please use arrow syntax for these functions as stated in the project requirements 👍

File: modules/book-store.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      bookContainer.innerHTML = `
+        <div class='book-list-container'>
+          <div class='wrap'>
+            <div>
+                <p>${book.titleBook}</p>
+                <p> ${book.authorBook}</p>
+            </div>
+            <div class='removeBtn'>
+              <button onclick="removeBook(${index})" class='delete'>Remove</button>
+            </div>
```
Comment (codepantha) [inline modules/book-store.js]: - [ ] An error occurs when trying to remove a book. Please see the error below and try to fix it 👍 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @sulaimonaa 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used ES6 and modules 📌 
✔️ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @sulaimonaa 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Fixed remove errors :+1: 
Used arrow functions :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**